content/en/docs/instrumentation/ruby/manual.md

```diff
@@ -103,40 +230,139 @@ else
 end
 ```
 
-A useful characteristic of events is that their timestamps are displayed as offsets from the beginning of the span, allowing you to easily see how much time elapsed between them.
+A useful characteristic of events is that their timestamps are displayed as
+offsets from the beginning of the span, allowing you to easily see how much time
+elapsed between them.
 
 Events can also have attributes of their own e.g.
 
 ```ruby
-span.add_event("Cancelled wait due to external signal", attributes: { "pid" => 4328, "signal" => "SIGHUP" })
+require "opentelemetry/sdk"
+
+span.add_event("Cancelled wait due to external signal",
+  attributes: { "pid" => 4328, "signal" => "SIGHUP" })
+```
+
+### Add Span Links
+
+A [span](/docs/concepts/signals/traces#spans-in-opentelemetry) can be created
+with zero or more [span links](/docs/concepts/signals/traces#span-links) that
+causally link it to another span. A link needs a [span
+context](/docs/concepts/signals/traces#span-context) to be created.
+
+```ruby
+require "opentelemetry/sdk"
+
+span_to_link_from = OpenTelemetry::Trace.current_span
+
+link = OpenTelemetry::Trace::Link.new(span_to_link_from.context)
+
+MyAppTracer.in_span("new-span", links: [link])
+  # do something that 'new_span' tracks
+
+  # The link in 'new_span' casually associated it with the previous one,
+  # but it is not a child span.
+end
+```
+
+Span Links are often used to link together different traces that are related in
+some way, such as a long-running task that calls into sub-tasks asynchronously.
+
+Links can also be created with additional attributes:
+
+```ruby
+link = OpenTelemetry::Trace::Link.new(span_to_link_from.context, attributes: { "some.attribute" => 12 })
+```
+
+### Set span status
+
+A [status](/docs/concepts/signals/traces#span-status) can be set on a
+[span](/docs/concepts/signals/traces#spans-in-opentelemetry), typically used to
+specify that a span has not completed successfully - StatusCode.ERROR. In rare
+scenarios, you could override the Error status with StatusCode.OK, but don’t set
+StatusCode.OK on successfully-completed spans.
+
+The status can be set at any time before the span is finished:
+
+```ruby
+require "opentelemetry/sdk"
+
+current_span = OpenTelemetry::Trace.current_span
+
+begin
+   1/0 # something that obviously fails
+rescue
+    current_span.status = OpenTelemetry::Trace::Status.error("error message here!")
+end
```

Comment:
```suggestion
begin
  1/0 # something that obviously fails
rescue
  current_span.status = OpenTelemetry::Trace::Status.error("error message here!")
end
```

The `in_span` helper will set status and record exceptions for you; I'm not sure if that's worth mentioning here or not.